src/test/resources/mockData/generatedBinaryFile.txt

```diff
@@ -0,0 +1 @@
+EMPTY FILE THAT IS GENERATED WHILE EXPORT FOR THE TESTING NEEDS
```

Comment:
I know this is just a mock file, but please use allowable file extensions and formats.

Reply:
Sure.